torch/testing/_internal/common_nn.py

```diff
@@ -1674,6 +1674,19 @@ def fractional_max_pool3d_test(test_case):
         desc='2d_affine_large_feature',
         test_cpu=False,
     ),
+    dict(
+        module_name='GroupNorm',
+        constructor_args=(3, 12, 1e-3),
+        cpp_constructor_args='torch::nn::GroupNormOptions(3, 6).eps(1e-3)',
+        input_size=(4, 12, 28, 28),
+        cudnn=True,
+        check_eval=True,
+        check_bfloat16=True,
+        desc='2d_no_affine_large_feature_forward',
+        check_forward_only=True,
+        reference_fn=lambda x, *_: torch.nn.functional.layer_norm(
+            x.view(4, 3, -1), (4 * 28 * 28,), eps=1e-3).view(4, 12, 28, 28)
```

Comment:
is this size enough to observe better accuracy (i.e. will it fail without this PR?). Also, IIRC you never landed LayerNorm better accuracy PRs, so using LayerNorm as reference could be inaccurate?

Reply:
The original purpose of this test is to make sure we cover the test cases larger than the chunksize. It is a little hard to have a reliable test case for the numerical issue. If the input size is large, the test will be pretty slow. I removed the layer_norm and used torch.mean and torch.std for reference_fn.